src/lib/traitify.js

```diff
@@ -1,6 +1,14 @@
-import Traitify from "traitify-client";
-Traitify.ui = require("traitify-ui").default;
-Traitify.ui.client = Traitify;
-Traitify.__version__ = __VERSION__;
+
+/*global __VERSION__ */
+import TraitifyLib from "traitify-client";
 require("polyfills");
-export default Traitify;
\ No newline at end of file
+import ui from "traitify-ui";
+function Init(){
```

Comment:
Looks like in another place you export the Init function, but here you export the the result of the Init function. What's the purpose of either? I don't have much experience around this so I just wanted to know